exercises/roman-numerals/src/example.c

```diff
@@ -0,0 +1,43 @@
+#include "roman_numerals.h"
+#include <stdlib.h>
+#include <string.h>
+
+#define NUM_OF_ELEMENTS(a)    (sizeof(a) / sizeof(a[0]))
+#define MAX_NUMERAL_LENGTH    (6)
+
+typedef struct
+{
+    char *numeral;
+    unsigned int value;
+} Numeral_Values_t;
+
+const Numeral_Values_t numeral_values[] =
+{
+    { "M", 1000 },
+    { "CM", 900 },
+    { "D",  500 },
+    { "CD", 400 },
+    { "C",  100 },
+    { "XC",  90 },
+    { "L",   50 },
+    { "XL",  40 },
+    { "X",   10 },
+    { "IX",   9 },
+    { "V",    5 },
+    { "IV",   4 },
+    { "I",    1 }
+};
+
+char * to_roman_numeral(unsigned int number)
+{
+   char *numerals = calloc(0, sizeof(char) * MAX_NUMERAL_LENGTH);
+
+   for (size_t i = 0; i < NUM_OF_ELEMENTS(numeral_values); i++) {
+      while (number >= numeral_values[i].value) {
+         strcat(numerals, numeral_values[i].numeral);
```

Comment:
Changed to strncat although I'm questioning my calculation of the length.  Let me know what you guys think.